### PR TITLE
fix(argo-cd): add functionality to en/disable argocd-ssh-known-hosts-cm

### PR DESCRIPTION
**Pull request by gajicdev**
Checklist:

* [x] I have bumped the chart version according to [versioning](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#versioning)
* [x] I have updated the documentation according to [documentation](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#documentation)
* [x] I have updated the chart changelog with all the changes that come with this pull request according to [changelog](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#changelog).
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/blob/master/community/CONTRIBUTING.md).
* [x] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/stable/developer-guide/ci/)).

Closes argoproj/argo-helm#3082

### CODE REVIEW

File: charts/argo-cd/values.yaml [hunk 1]
```diff
@@ -371,6 +371,9 @@ configs:
   # SSH known hosts for Git repositories
   ## Ref: https://argo-cd.readthedocs.io/en/stable/operator-manual/declarative-setup/#ssh-known-host-public-keys
   ssh:
+    # -- Specifies if the argocd-ssh-known-hosts-cm configmap should be created by Helm.
+    enabled: true
```
Comment (mkilchhofer) [inline charts/argo-cd/values.yaml:375]: (Question zo other maintainers):
Do we wanna use `create` as a keyword here to be inline with other ConfigMaps conditionals?

/cc: @yu-croco , @tico24 , @mbevc1

**Review by mkilchhofer — COMMENTED** [review]
Hi @gajicdev 

thanks for your contribution 🙏 .
Only found 2 small things.

Regarding the "correct" keyword, I'd like to have feedback from other maintainers

File: charts/argo-cd/values.yaml (hunk 1 above)
Comment (yu-croco) [reply charts/argo-cd/values.yaml:375]: I vote to `create` to align existing ones. 😀

Comment (mbevc1) [reply charts/argo-cd/values.yaml:375]: I concur with @yu-croco

Comment (gajicdev) [reply charts/argo-cd/values.yaml]: changed it to `create`

File: charts/argo-cd/Chart.yaml [hunk 2]
```diff
@@ -28,3 +28,5 @@ annotations:
   artifacthub.io/changes: |
     - kind: changed
       description: Bump argo-cd to v2.13.2
```
[the remark this answers is not in the document]
Comment (gajicdev) [reply charts/argo-cd/Chart.yaml]: removed the old lines 👍

**Review by yu-croco — APPROVED** [review]

**Review by mkilchhofer — APPROVED** [review]
LGTM
Again, thanks for your contribution @gajicdev 🙏

**Review by mbevc1 — APPROVED** [review]

**Pull request merged**